backends/memcheck/ceed-memcheck-qfunction.c

```diff
@@ -63,8 +63,12 @@ static int CeedQFunctionApply_Memcheck(CeedQFunction qf, CeedInt Q, CeedVector *
     // Note: need field size because vector may be longer than needed for output
     CeedCallBackend(CeedQFunctionFieldGetSize(output_fields[i], &field_size));
     for (CeedSize j = 0; j < Q * field_size; j++) {
-      CeedCheck(!isnan(impl->outputs[i][j]), ceed, CEED_ERROR_BACKEND, "QFunction output %d entry %ld is NaN after restoring write-only access", i,
-                j);
+      char *kernel_path, *kernel_name;
```

Comment:
hmm, now I'm wondering if these should be `const` parameters

Reply:
Yeah, I'd probably say so. I can't think of any reason that they should ever need to change after the QFunction has been created.

Reply:
cool, I'll investigate in a follow-up